### PR TITLE
[WIP] Landingpage Overhaul

### PR DESCRIPTION
**Pull request by Tzky**
I felt that the background images made the index look rather messy. This is an attempt to fix it.

Preview:
https://htmlpreview.github.io/?https://github.com/SpongePowered/SpongeHome/blob/tzk-adjustments/index.html

### CODE REVIEW

**Comment by jamierocks** [comment]
The pictures aren't centred with their text -

File: index.html [hunk 1]
```diff
@@ -98,96 +98,55 @@
                 <div class="col-lg-12">
                     <img class="img-responsive" src="assets/img/icons/spongie-mark-dark.png" alt="">
                     <div class="intro-text">
-                        <span class="name">SpongePowered</span>
+                        <span class="name">Sponge</span>
                         <!-- <hr class="star-light"> -->
-                        <span class="skills">A New Minecraft Modding API</span>
+                        <span class="skills">The New Minecraft Plugin API</span>
```
Comment (jamierocks) [inline index.html]: That makes it sound too official - also you need to change the title to plugin API

File: index.html [hunk 2]
```diff
@@ -98,96 +98,55 @@
...
-                        <br>
-                        <h4><a href="https://forums.spongepowered.org/t/psa-spongeapi-4-0-and-what-it-means-for-you/11728">Continue Reading</a></h4>
-                    </div>
-                </div>
-            </div>
+    <!-- Quicklinks Section -->
+    <section class="offsite-links" id="offsite-links">
+      <div class="container">
+        <div class="row">
+          <div class="col-xs-2"><a href="https://forums.spongepowered.org/"><i class="icon-pencil"></i><p>Forums</p></a></div>
```
Comment (jamierocks) [inline index.html]: `<div class="col-xs-2 text-center">` should do

Comment (Tzky) [inline index.html]: yep, that worked

File: index.html [hunk 3]
```diff
@@ -302,21 +297,22 @@
                 </div>
             </div>
             <div id="socialIcons">
-                <a href="https://twitter.com/spongepowered">
-                    <img src="assets/img/icons/twitter.png" class="img-responsive" alt="Twitter">
```
Comment (simon816) [inline index.html:306]: Your rebase overwrote this
Edit: fixed

**Comment by felixoi** [comment]
I think either the background of about or of social media should be changed

File: index.html [hunk 4]
```diff
@@ -309,7 +309,7 @@
                         <img src="assets/img/icons/reddit.svg" class="img-responsive" alt="Reddit">
                     </a>
 
-                    <a href="https://www.reddit.com/r/spongeproject">
+                    <a href="https://www.twitch.tv/spongepowered">
```
Comment (MichaelHillcox) [inline index.html]: Think this was fixed. You are on a old branch

Comment (Tzky) [inline index.html]: Looks like i messed the branch up while resolving merge conflicts... ;)

File: index.html [hunk 5]
```diff
@@ -98,96 +98,55 @@
...
     </section>
 
     <!-- Downloads Section -->
     <section class="downloads" id="downloads">
         <div class="container">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h2>Downloads</h2>
-                    <h1><br></h1>
+                    <h2>Get SpongePowered</h2>
```
Comment (stephan-gh) [inline index.html]: Shouldn't that be "Get Sponge"?

Comment (Tzky) [inline index.html]: that was intentional, but i'll revert as i prefer "get Sponge" too.

**Comment by Tzky** [comment]
@Feeliiix done, social media now has a light background

**Comment by felixoi** [comment]
@Tzky Yep, i think that looks better.

Some things:
- Icon positions at start and social icon positions are buggy at mobile screens
- I would change the Get Sponge images. I think all would be cleaner if no one of this have the Sponge logo and the 2nd line is centered
- Colored social icons (because of the light background)
- "Announcements" headline is buggy at mobile screens

I think all these things don't have to be finished in 1 PR.

**Comment by stephan-gh** [comment]
The new buttons looksp kind of weird on smaller screen sizes for me

**Comment by Tzky** [comment]
i haven't tried this on smaller screens (yet).

**Comment by Tzky** [comment]
Fixed the download buttons. I think this PR is ok for now. Any further input?
@Feeliiix Yes, we can fix most of this in other PRs.

@Zidane 
Is this PR a step in the right direction? If so, i'll squash and merge.

Preview:
https://htmlpreview.github.io/?https://github.com/SpongePowered/SpongeHome/blob/tzk-adjustments/index.html

**Comment by felixoi** [comment]
Yep, i have some time today. I'm going to fix the mobile things

**Pull request merged**